### PR TITLE
docs: github oauth support

### PR DESCRIPTION
**Pull request by misterbisson**

### CODE REVIEW

File: content/intro/getting-started/index.md [hunk 1]
```diff
@@ -11,73 +11,43 @@ tags:
...
-Once you have signed up, you can immediately start scanning repositories and commits for vulnerabilities.
+1. Sign up
+2. Add a code server
+3. Start improving security
 
-If you are ready to improve your code's security, please read below to get started for free.
+## Signing up
 
-## Signing up with BluBracket
+To start using BluBracket for free, visit [our signup page](https://blubracket.com/contact/get-started/) and click the **call to action** button.
```
Comment (misterbisson) [inline content/intro/getting-started/index.md:29]: **call to action** needs to be replaced when that text is decided.

File: content/intro/getting-started/index.md [hunk 2]
```diff
@@ -11,73 +11,43 @@ tags:
...
 
-To create a user account, visit [https://blubracket.com/contact/get-started/](https://blubracket.com/contact/get-started/). Click the **Start for Free** button.
+{{< figure src="three-step-process-sign-up.png" caption="Begin your three-step process by clicking the call to action. ">}}
```
Comment (misterbisson) [inline content/intro/getting-started/index.md:31]: This image needs to be replaced when we've decided the CTA text.

File: content/intro/getting-started/index.md [hunk 3]
```diff
@@ -11,73 +11,43 @@ tags:
 searchExtraKeywords:
   - "Getting Started with BluBracket Teams and Enterprise Editions - BluBracket"
   - "Getting Started with the Free Trial - BluBracket"
+  - oauth
+  - GitHub
 resources:
   - src: three-step-process-sign-up.png
```
Comment (misterbisson) [inline content/intro/getting-started/index.md:17]: This image needs to be replaced when we've decided the CTA text.

**Review by robbyandrews — DISMISSED** [review]

File: content/intro/getting-started/index.md [hunk 4]
```diff
@@ -8,76 +8,51 @@ weight: 300
...
-
-{{< figure src="set-your-blubracket-password.jpg" caption="Follow the password requirements to set your new password for your BluBracket account." >}}
-
-Fill in your email address and password.
-
-**Password criteria**:
-
-- Passwords must be at least eight characters.
-- Passwords must include at least one number, a special character or symbol, and one uppercase letter.
+Bookmark your account so you can come back easily.
```
Comment (srinivasakartheek) [inline content/intro/getting-started/index.md:40]: nit: Can we highlight this line?

**Review by srinivasakartheek — APPROVED** [review]

**Pull request merged**